firebase-firestore/src/test/java/com/google/firebase/firestore/model/FieldValueTest.java

```diff
@@ -356,4 +374,12 @@ public void testValueOrdering() {
         .addEqualityGroup(wrapObject(map("foo", "0")))
         .testCompare();
   }
+
+  private ObjectValue setField(ObjectValue empty, String a, FieldValue mod) {
```

Comment:
The caller doesn't necessarily have to pass an empty object, do they? Seems like `empty` could be `objectValue`, as it is below.

Reply:
Yeah, it should have been objectValue.